editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -1348,13 +1350,9 @@ EditorAssetLibrary::EditorAssetLibrary(bool p_templates_only) {
 	search_hb2->add_child(memnew(Label(TTR("Site:") + " ")));
 	repository = memnew(OptionButton);
 
-	// FIXME: Reenable me once GH-7147 is fixed.
-	/*
 	repository->add_item("godotengine.org");
 	repository->set_item_metadata(0, "https://godotengine.org/asset-library/api");
-	*/
-	repository->add_item("localhost");
-	repository->set_item_metadata(/*1*/ 0, "http://127.0.0.1/asset-library/api");
```

Comment:
You should leave localhost, it is useful when testing the assetlib (also, uncomment the `1` there).

Reply:
Ok

Reply:
Check ?

Reply:
Ah, you write uncommenting "1" ? Ok..

Reply:
It should not be commented out, it should be active, with id `1` instead of `0`.

Reply:
Ok

Reply:
repository->set_item_metadata(0, "https://godotengine.org/asset-library/api");

should also be 1 instead 0 ?

Reply:
It should be:
```
repository->add_item("godotengine.org");
repository->set_item_metadata(0, "https://godotengine.org/asset-library/api");
repository->add_item("localhost");
repository->set_item_metadata(1, "http://127.0.0.1/asset-library/api");
```

Reply:
Ah.. ok